### PR TITLE
get api users endpoint added

### PR DESCRIPTION
**Pull request by Antony-Lester**
also removed unneeded files, controller.js, model.js

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const request = require('supertest');
 const db = require(`../db/connection`);
 const seed = require('../db/seeds/seed');
 const testData = require('../db/data/test-data');
+const { expect } = require('@jest/globals');
```
Comment (alexdevdavis) [inline __tests__/app.test.js:6]: was this an auto-complete from VSCode?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -84,4 +84,39 @@ describe('get:', () => {
...
+                .then(res => {
+                    expect(res._body).toBeInstanceOf(Array);
+                });
+        });
+        test('status:200 returns a array of objects containing property: username, name & avatar_url', () => {
+            return request(app)
+                .get('/api/users')
+                .expect(200)
+                .then(res => {
+                    res._body.forEach(news => {
```
Comment (alexdevdavis) [inline __tests__/app.test.js:101]: suggest doing a length check before forEach to ensure you're not yielding a false positive

**Review by alexdevdavis — APPROVED** [review]
Happy to approve this, just add the .length check before your forEach

File: __tests__/app.test.js (hunk 1 above)
Comment (Antony-Lester) [reply __tests__/app.test.js:6]: unfortunately yes

**Pull request merged**